docs/user-guide/user-manual/detailed-itineraries.md

```diff
@@ -115,11 +114,11 @@ destinations = geopandas.GeoDataFrame(
 
 detailed_itineraries_computer = r5py.DetailedItinerariesComputer(
     transport_network,
-    origins=origins,
-    destinations=destinations,
-    departure=datetime.datetime(2022,2,22,8,30),
-    transport_modes=[r5py.TransportMode.TRANSIT, r5py.TransportMode.WALK],
-    snap_to_network=True,
+    origins = origins,
+    destinations = destinations,
+    departure = datetime.datetime(2022,2,22,8,30),
+    transport_modes = [r5py.TransportMode.TRANSIT, r5py.TransportMode.WALK],
+    snap_to_network = True,
```

Comment:
This is different style in R than in Python, see https://peps.python.org/pep-0008/#other-recommendations

Reply:
Oh, Ok. I thought it would help readability and thought it would be Ok since there were some spaces in other parts.